crates/turborepo-lib/src/hash/mod.rs

```diff
@@ -130,7 +130,9 @@ impl From<LockFilePackages> for Builder<HeapAllocator> {
             }
         }
 
-        let mut canon_builder = Builder::new(HeapAllocator::default());
+        let size = builder.total_size().unwrap().word_count + 1;
```

Comment:
Can you handle the error or leave a comment as to why the `.unwrap()` is safe?

Reply:
We already have multiple unwraps in this code. We could switch this to a `TryFrom`, but to be honest, if this code fails, it's a fatal programmer error, so I don't feel too bad about unwrapping here. I'll switch it to an `expect`.